packages/wallet/backend/src/rapyd/rapyd.d.ts

```diff
@@ -325,4 +325,24 @@ interface CompletePayoutRequest {
   amount: number
 }
 
+interface DailyRateRequest {
+  action_type: string
+  amount?: number
+  buy_currency: string
+  date?: string
+  fixed_side?: string
+  sell_currency: string
+}
+
+interface Rate {
+  sell_currency: string
```

Comment:
@dragosp1011 for this Rate object, this is response from Rapyd. 

Do you think it's necessary to create one more interface for otherRate object with camelcase, and convert?

Reply:
this is my opinion, maybe @beniaminmunteanu and @raducristianpopa have other thoughts

Reply:
Given the fact that these interfaces and types will be moved to zod schemas when I get to finish up on the rapyd future proof PR, I think this can remain as it is in this PR.

Reply:
Got it. Then I can mark this as 'resolved'?